src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.cpp

```diff
@@ -2175,6 +2175,10 @@ bsl::shared_ptr<mqbi::Queue> ClusterQueueHelper::createQueueFactory(
                 context.d_domain_p);
         }
 
+        BALL_LOG_ERROR << d_cluster_p->description()
```

Comment:
Why is this log at ERROR level? Should be at INFO level.

Reply:
Good catch, thank you.  Removing this leftover